packages/dynamite/dynamite/lib/src/builder/client.dart

```diff
@@ -472,15 +473,18 @@ String buildParameterSerialization(
   final $default = parameter.schema?.$default;
   var defaultValueCode = $default?.value;
   if ($default != null && $default.isString) {
-    defaultValueCode = "'${$default.asString}'";
+    defaultValueCode = escapeDartString($default.asString);
   }
   final dartName = toDartName(parameter.name);
   final serializedName = '\$$dartName';
-
-  final buffer = StringBuffer()..write('var $serializedName = ${result.serialize(dartName)};');
+  final buffer = StringBuffer();
 
   if ($default != null) {
-    buffer.writeln('$serializedName ??= $defaultValueCode;');
+    buffer
+      ..write('var $serializedName = ${result.serialize(dartName)};')
+      ..writeln('$serializedName ??= $defaultValueCode;');
```

Comment:
Why not directly use ?? and make the variable final

Reply:
I think the output is way cleaner (and we already do so).